File: C/Lista_03(Estrutura_Repetitiva)/Media_Ponderada.c
#include <stdio.h>

int main () {

    int n;

    printf("Quantos casos voce vai digitar? ");
    scanf("%d", &n);

    for(int i=0;i<n;i++){
        double x, y, z;
        printf("Digite tres numeros:\n");
        scanf("%lf", &x);
        scanf("%lf", &y);
        scanf("%lf", &z);

        double media = (2*x+3*y+5*z)/10;

        printf("MEDIA = %.1lf\n", media);
    }

    return 0;

}

File: Python/Lista_03(Estrutura_Repetitiva)/Media_Ponderada.py
n = int(input("Quantos casos voce vai digitar? "))

for i in range(0,n):
    print("Digite tres numeros:")
    n1 = float(input())
    n2 = float(input())
    n3 = float(input())
    media = (2*n1+3*n2+5*n3)/10
    print(f"MEDIA = {media:.1f}")